0029-loopback-transport/0029-loopback-transport.md

```diff
@@ -0,0 +1,33 @@
+---
+title: The Loopback Transport Protocol (LT)
+draft: 1
+---
+# Loopback Transport Protocol (LT)
+
+The Loopback Transport protocol is a technique for transporting value over the Interledger. It relies on the establishment of a temporary, direct, ILPv4 link between sender and receiver (the loopback link). The flow is as follows.
```

Comment:
Can the BTP link be used for sending data as well? If so, please describe how that works

Reply:
it's described in the BTP spec, just set `protocolName` to something other than `'ilp'`

Reply:
Is choosing that protocolName and message format in or out of scope for this protocol?

Reply:
out of scope, but it could be part of the setup protocol

Reply:
Could you please make a list of the inputs to this protocol (the things that need to be negotiated or communicated up front) like there is in the [PSK2 spec](https://github.com/interledger/rfcs/blob/master/0025-pre-shared-key-2/0025-pre-shared-key-2.md#setup)?

Reply:
see https://github.com/interledger/rfcs/pull/413/files#diff-33e3b9e69eaad2fce9a64285ee83f39dR35 (follow-up PR to this one, which deals with the setup flow)